changes/b4642a502f49353787b4f6632a6a6566.yaml

```diff
@@ -0,0 +1,9 @@
+---
+desc: Fixed an issue with the Storm loop and generator keywords, ``continue``, ``break``,
+  and ``stop``. Using these keywords to exit a function will now raise a ``StormRuntimeError``
```

Comment:
Isn't this more like "using these keywords outside of a loop/generator will now raise"?